beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1686,13 +1686,13 @@ pub static DATA_COLUMN_SIDECAR_GOSSIP_VERIFICATION_TIMES: LazyLock<Result<Histog
             "Full runtime of data column sidecars gossip verification",
         )
     });
-pub static DATA_COLUMNS_SIDECAR_PROCESSING_SUCCESSES: LazyLock<Result<IntCounter>> =
-    LazyLock::new(|| {
-        try_create_int_counter(
-            "beacon_blobs_column_sidecar_processing_successes_total",
-            "Number of data column sidecars verified for gossip",
-        )
-    });
+
+pub static CUSTODY_COLUMNS_COUNT: LazyLock<Result<IntGauge>> = LazyLock::new(|| {
+    try_create_int_gauge(
+        "beacon_custody_columns_count_total",
+        "Total count of columns in custody",
```

Comment:
It might be worth clarifying whether this number refers to the number of columns in custody (up to 128), or the total number of columns in custody within the data availability boundary. I'm guessing this means the latter?

Reply:
It would be the second,  it would seesaw as new columns come into custody and old ones are pruned.